pages/docs/index.js

```diff
@@ -42,7 +42,7 @@ const SubHeader = styled.h3`
 `
 
 const Documentation = () => (
-  <DocsLayout title="Documentation">
+  <DocsLayout title="Documentation" description="Get the best out of styled components">
```

Comment:
This sounds a tad weird. We've previously just used:

> Use the best bits of ES6 and CSS to style your apps without stress 💅

But here I'd go for something really explicit like:

> Learn how to use styled-components and to style your apps without stress

Reply:
Would be great if you can give me the descriptions for all the pages. :)